test/test_namedtensor.py

```diff
@@ -85,6 +85,18 @@ def test_has_names(self):
         self.assertTrue(partially_named.has_names())
         self.assertTrue(fully_named.has_names())
 
+    def test_repr(self):
+        named_tensor = torch.zeros(2, 3).set_names_(['N', 'C'])
```

Comment:
It's not really related to this particular PR, but do we have other tensor functions named this way? To me the `set` and trailing `_` seem to redundantly express the same thing. I feel like we could have a better name for it.

Reply:
We have:
- set_data(...)
- set_
- set_quantizer_
so an example and a counter-example. I'm taking suggestions for a better name!